Purple/index.css

```diff
@@ -0,0 +1,108 @@
+@import url('https://fonts.googleapis.com/css2?family=Ubuntu&display=swap');
+
+body,html{
+    padding: 0;
+    margin: 0;
+    overflow: hidden;
+    font-family: "Ubuntu", sans-serif;
+}
+
+.ifFcpp{
+    position: absolute;
+    background: rgb(148, 42, 255);
+    box-shadow: 0px 0px 8px rgb(148, 42, 255);
+    border-radius: 100px;
+    color: rgba(255, 255, 255, 0.45);
+    font-size: 18px;
+    padding: 5px;
+    top:10px;
+    left: 330px;
+    transform: translateX(-50%);
+    transition: 300ms ease-in-out;
+    opacity: 1;
+}
+#ifFcpp{
+    color: white;
+}
+
+.base{
+    position: absolute;
+    height: 49px;
+    width: 340px;
+    border-radius: 100px;
+    background: rgb(54, 5, 104);
+    top: 25px;
+}
```

Comment:
my recommendation is to fix css to make indents little bit smother

.base {
  position: absolute;
  height: 50px;
  width: 340px;
  border-radius: 25px;
  background: rgb(54, 5, 104);
}